### ISSUE TITLE
dist missing

### ISSUE THREAD

**Issue by panuhorsmalahti** [coveo/exponential-backoff#7]
It seems like versions 1.2.0 and 2.0.0 are missing the dist folder in the npm package.

```
temp $ npm install exponential-backoff@1.0.7
+ exponential-backoff@1.0.7
added 1 package from 1 contributor and audited 1 package in 1.945s
found 0 vulnerabilities

temp $ cd node_modules/exponential-backoff/
exponential-backoff $ ls
LICENSE		README.md	dist		package.json
exponential-backoff $ cd ..
node_modules $ cd ..
temp $ rm -rf node_modules/
temp $ npm install exponential-backoff@1.2.0
+ exponential-backoff@1.2.0
added 1 package from 1 contributor and audited 1 package in 0.664s
found 0 vulnerabilities
temp $ cd node_modules/exponential-backoff/
exponential-backoff $ ls
LICENSE		README.md	package.json
exponential-backoff $ cd ..
node_modules $ cd ..
temp $ rm -rf node_modules/
temp $ npm install exponential-backoff@2.0.0
+ exponential-backoff@2.0.0
added 1 package from 1 contributor and audited 1 package in 0.693s
found 0 vulnerabilities

temp $ cd node_modules/exponential-backoff/
exponential-backoff $ ls
LICENSE		README.md	doc		package.json
```

**Comment by samisayegh** [comment]
I will take a look this weekend 🤦‍♂️ .

**Comment by samisayegh** [comment]
The issue was I did not readd the `skip_cleanup` option after rehashing the npm token inside the travis yaml file. Should be fixed with version [2.1.0](https://www.npmjs.com/package/exponential-backoff/v/2.1.0). 

Thanks for reporting,
